CONTIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to wild
+
+If you'd like to help out, I'd love to hear from you. It's a good idea to reach out first to avoid
+duplication of effort. Also, it'll make it possible for me to provide hints that might make what
+you're trying to do easier.
+
+## Options for communicating
+
+* I like, where possible, to talk to people via video chat.
+* You can book a time in my [calendar](https://calendar.app.google/MBYQeATMNBvuK8AZ6). If time zones make this hard, let
+  me
```

Comment:
Line wrapping looks weird here. I generally use 100 column wrap. Although, I'm also happy to just clean up line wrapping next time I'm editing the document if you want to leave it as-is.

Reply:
Fixed (to 80 cols)